test/mocha.opts

```diff
@@ -2,6 +2,7 @@ test/helpers/*.js
 test
 --compilers js:babel-core/register
 --require ignore-styles
+--require babel-polyfill
```

Comment:
Should not include babel-polyfill for tests as our server side code should never require any form of transpiling.